### ISSUE TITLE
Creating proposal vote should not delete global delegation

### ISSUE THREAD

**Issue by jinjagit** [liquidvotingio/api#161]
Currently, creating a proposal-specific vote deletes a global delegation (as the `proposal_url` is not used to search for a 'conflicting' delegation, in `voting.ex/create_vote/1`). Only a proposal-specific delegation for the same proposal as a proposal-specific vote, should be deleted on vote creation.

**Comment by oliverbarnes** [comment]
Is this fixed?

**Comment by jinjagit** [comment]
Hmm. I _think_ so. Lemme check (I'll do it now and confirm here)

**Comment by jinjagit** [comment]
Yes fixed. Trying to create a proposal-specific delegation when a conflicting global delegation exists, results in a no-op + 'error' returned in GraphQL:
```
{
  "data": {
    "createDelegation": null
  },
  "errors": [
    {
      "details": "A global delegation for the same participants already exists.",
      "locations": [
        {
          "column": 3,
          "line": 2
        }
      ],
      "message": "Could not create delegation.",
      "path": [
        "createDelegation"
      ]
    }
  ]
}
```

I'll now check we have a test of that (pretty sure we do)

**Comment by jinjagit** [comment]
Confirmed we also have a test: `test/liquid_voting/delegations_test.exs` contains `test "upsert_delegation/1 with proposal-specifc data returns error if global delegation for same delegator/delegate pair exists"`

**Comment by jinjagit** [comment]
Closing as was resolved by PR #133
